docs/UserGuide.md

```diff
@@ -423,7 +425,8 @@ How this command works:
   In other words, the new `medicalHistory` must be different from the existing `medicalHistory`.
 
 Examples:
-* `medical 1 m/ADHD`
+* `medical 1 m/ADHD` adds ADHD to the `medicalHistory` of student 1 in the displayed student list.
+* `medical 1 m/``removes the `medicalHistory` from student 1 in the displayed student list.
```

Comment:
@nniiggeell , could  you change all `medicalHistory` to `MEDICAL_HISTORY` so that it's consistent with the parameter?

Reply:
Ok.